### PR TITLE
Fix: add missing keys that eslint was complaining about

### PR DESCRIPTION
**Pull request by karliatto**
## Description

Add missing keys to iterator to satisfy eslint

## Screenshots:
image

### CODE REVIEW

File: suite-common/message-system/src/messageSystemThunks.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import {
     selectMessageSystemTimestamp,
     selectMessageSystemCurrentSequence,
 } from './messageSystemSelectors';
+// eslint-disable-next-line import/extensions
```
Comment (Nodonisko) [inline suite-common/message-system/src/messageSystemThunks.ts]: Please rebase and remove this disable. I already fixed this in global config.

**Review by Nodonisko — APPROVED** [review]
Thanks for fixing that warnings 🚀

File: packages/blockchain-link/tests/integration/worker/index.ts [hunk 2]
```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-var-requires,import/no-extraneous-dependencies */
```
Comment (Nodonisko) [inline packages/blockchain-link/tests/integration/worker/index.ts:1]: Only `@typescript-eslint/no-var-requires` part should have been removed

File: suite-common/message-system/src/messageSystemThunks.ts (hunk 1 above)
Comment (karliatto) [reply suite-common/message-system/src/messageSystemThunks.ts]: It works for me locally, no issues when I run `yarn lint` but here in GitHub job fails:

image

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**